Sources/Purchasing/Configuration.swift

```diff
@@ -129,9 +129,23 @@ import Foundation
          * Set `observerMode`.
          * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
          * RevenueCat's backend. Default is `false`.
+         *
+         * - Note: This assumes your IAP implementation uses StoreKit 1.
+         * If you use StoreKit 2, use ``with(observerMode:storeKit2:)`` instead.
```

Comment:
I wonder if we should deprecate this method and ask people to use the other one to make it more explicit...

Reply:
I think that would be a good way to notify developers that are currently using this function

Reply:
Yeah I'll do that.

Reply:
Done.

Reply:
If a developer is _currently_ using this function, it's almost certain that they're using SK1, though, right? Like, it seems fairly unlikely that you: 
- have an existing SK1 implementation that you don't want to touch, but you want RC, so you add observer mode
- decide that you _do_ want to move into SK2, but still want to keep RC in observer mode

Right? 

So I'd vote to just keep a single method with a default to sk1